### PR TITLE
Add calculated_reach_length trigger + update slope calculation

### PR DESCRIPTION
**Pull request by varrieta**
Add a before trigger on qgep.od_reach to calculate the length and save it into length_effective

 Modify slope calculation in vw_qgep_reach to avoid division by zero that makes the database crashes

### CODE REVIEW

**Comment by varrieta** [comment]
@m-kuhn This time it worked :)

**Comment by varrieta** [comment]
If it works like that, can anyone commit? :)

File: 06_symbology_functions.sql [hunk 1]
```diff
@@ -392,6 +392,9 @@ BEGIN
     WHEN TG_OP = 'DELETE' THEN
       rp_obj_id = OLD.obj_id;
   END CASE;
+  
+  UPDATE qgep.od_reach
```
Comment (m-kuhn) [inline 06_symbology_functions.sql]: Can you add a comment here that explains why this is done (i.e. so we know in 5 years still, that this is done to retrigger the `calculate_reach_length` trigger on update)

**Comment by m-kuhn** [comment]
Good stuff, would be great to add this comment and then ready to merge

**Comment by varrieta** [comment]
Just did, merci Matthias :)

**Comment by m-kuhn** [comment]
Merci Valentine

**Pull request merged**